numba/cpython/randomimpl.py

```diff
@@ -1595,7 +1595,7 @@ def noncentral_chisquare(df, nonc, size=None):
 
     @register_jitable
     def noncentral_chisquare_single(df, nonc):
-        # identical to numpy implementation 
+        # identical to numpy implementation from distributions.c
```

Comment:
Thank you for making this update. I think it would be useful to provide a full link to a stable github URL. I.e. one with line-numbers and a SHA in the URL. Thank you!

Reply:
Yep, not a problem. I've added it in.